detlab/logcryo.py

```diff
@@ -837,15 +869,27 @@ def logtemp(**tconfig):
         print( time.ctime(), "(main) ERROR: %s missing config key 'name'" % args.config_file )
         sys.exit(1)
 
+    if 'influxdb_url' in config and 'influxdb_token' in config and 'influxdb_org' in config:
+        print(time.ctime(), "Will connect to influxDB.")
+        config['influxdb_client'] = True
+    else:
+        print(time.ctime(), "Will NOT connect to influxDB.")
```

Comment:
"influxDB connection details incomplete" would be more informative

Reply:
Changed to:
`print(time.ctime(), "Insufficient connection parameters, Will NOT connect to influxDB.")`